ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/io/FileOperations.java

```diff
@@ -53,8 +53,10 @@
   private final Path exportRootDataDir;
   private final String distCpDoAsUser;
   private HiveConf hiveConf;
-  private final FileSystem dataFileSystem, exportFileSystem;
+  private final FileSystem exportFileSystem;
```

Comment:
exportFileSystem object also need retry.

Reply:
there are many i/o failure scenario ..i am not sure if we need to handle all as part of this patch